curve-ansible/rolling_update_curve.yml

```diff
@@ -33,6 +33,7 @@
   roles:
     - { role: install_package, package_name: etcd, tags: install_etcd }
     - { role: generate_config, template_name: etcd.conf.yml, conf_path: "{{ etcd_config_path }}", tags: update_etcd_config }
+    - { role: install_package, package_name: etcd-daemon, tags: install_etcd_daemon }
```

Comment:
ags:
    - etcd
    - never
    
should never be deleted?

Reply:
> ags:
> - etcd
> - never
> 
> should never be deleted?

Don't need, run this yml use '--tags install_etcd_daemon' is ok.